API/src/main/java/com/bgsoftware/superiorskyblock/api/island/bank/IslandBank.java

```diff
@@ -38,6 +38,16 @@
      */
     BankTransaction depositAdminMoney(CommandSender commandSender, BigDecimal amount);
 
+    /**
+     * Deposit money into the bank, without taking money from any player.
+     *
+     * @param commandSender The player that deposited the money.
+     * @param amount        The amount to deposit.
+     * @param save          Whenever to log the transaction or not.
+     * @return The transaction details.
+     */
+    BankTransaction depositAdminLogMoney(CommandSender commandSender, BigDecimal amount, boolean save);
```

Comment:
the parameter name "save" is not indicative enough.
You can use "saveTransaction" for example, or something like that.